Makefile

```
@@ -15,7 +15,7 @@ PYTHON ?= /usr/bin/env python
 SYSCONFDIR = $(shell if [ $(PREFIX) = /usr -o $(PREFIX) = /usr/local ]; then echo /etc; else echo $(PREFIX)/etc; fi)
 
 # set markdown input format to "markdown-smart" for pandoc version 2 and to "markdown" for pandoc prior to version 2
-MARKDOWN = $(shell if [ `pandoc -v | head -n1 | cut -d" " -f2 | head -c1` = "2" ]; then echo markdown-smart; else echo markdown; fi)
+MARKDOWN = $(shell if [ `pandoc -v | head -n1 | cut -d" " -f2 | head -c1` -ge "2" ]; then echo markdown-smart; else echo markdown; fi)
```

Obviously not related to the issue, but this can be done just in POSIX shell, under the assumption that the major version is the digit string at the start of the second word in the version:
```suggestion
MARKDOWN = $(shell printf 'markdown%s\n' "$(pandoc -v | { read -r _ x; x=${x%%[^0-9]*}; [ "${x:-0}" -ge 2 ] && printf '%s' "-smart"; })")
```